src/sidebar/services/api.js

```diff
@@ -93,17 +90,16 @@ function get(object, path) {
  * @param {Promise<RouteMap>} links - API route data from API index endpoint (`/api/`)
  * @param {string} route - The dotted path of the named API route (eg. `annotation.create`)
  * @param {APIMethodCallbacks} callbacks
- * @return {APICallFunction}
+ * @return {APICall<Record<string, any>, object, object>}
  */
 function createAPICall(
   links,
   route,
   { getAccessToken, getClientId, onRequestStarted, onRequestFinished }
 ) {
-  return function (params, data, options = {}) {
+  return (params, data) => {
```

Comment:
I coulda sworn you rooted out all of these non-arrow functions lately...

Reply:
Indeed I used ESLint to do an automated conversion. It is possible it couldn't determine that such a change was safe in this context. There are no references to `this` in the function, so it should be fine though.